src/scales/scale.time.js

```diff
@@ -36,10 +40,94 @@ module.exports = function(Chart) {
 			},
 		},
 		ticks: {
-			autoSkip: false
+			autoSkip: false,
+			mode: 'series',   // 'linear|series'
+			source: 'auto'    // 'auto|labels'
 		}
 	};
 
+	function buildLookupTable(ticks, min, max, linear) {
+		var ilen = ticks.length;
```

Comment:
No, it's for performance, caching the length of the array is faster depending on the JS engine. `ilen`, `jlen`, etc. is commonly use out there and since it's most of the time only used for that purpose, I prefer to not change it (I also like that it's short and also refer to the `i` var).